ui/src/components/sidebar/connectionStats.tsx

```diff
@@ -98,144 +59,133 @@ export default function ConnectionStatsSidebar() {
-                        data={createChartArray(inboundRtpStats, "framesPerSecond").map(
-                          x => {
-                            return {
-                              date: x.date,
-                              stat: x.stat ? x.stat : null,
-                            };
-                          },
-                        )}
-                        domain={[0, 80]}
-                        unit=" fps"
-                      />
-                    )}
-                  </div>
-                </GridCard>
+
+              {/* Video Group */}
+              <div className="space-y-3">
+                <SettingsSectionHeader
+                  title="Video"
+                  description="The video stream from the JetKVM to the client."
+                />
+
+                {/* RTP Jitter */}
+                <Metric
+                  title="Network Stability"
+                  badge="Jitter"
+                  badgeTheme="light"
+                  description="How steady the flow of inbound video packets is across the network."
+                  stream={inboundVideoRtpStats}
+                  metric="jitter"
+                  map={x => ({
+                    date: x.date,
+                    metric: x.metric != null ? Math.round(x.metric * 1000) : null,
+                  })}
+                  domain={[0, 10]}
+                  unit=" ms"
+                />
+
+                {/* Playback Delay */}
+                <Metric
+                  title="Playback Delay"
+                  description="Delay added by the jitter buffer to smooth playback when frames arrive unevenly."
+                  badge="Jitter Buffer Avg. Delay"
+                  badgeTheme="light"
+                  data={jitterBufferAvgDelayData}
+                  gate={inboundVideoRtpStats}
+                  supported={
+                    someIterable(
+                      inboundVideoRtpStats,
+                      ([, x]) => x.jitterBufferDelay != null,
+                    ) &&
+                    someIterable(
+                      inboundVideoRtpStats,
+                      ([, x]) => x.jitterBufferEmittedCount != null,
+                    )
+                  }
+                  domain={[0, 30]}
+                  unit=" ms"
+                />
+
+                {/* Packets Lost */}
+                <Metric
+                  title="Packets Lost"
+                  description="Count of lost inbound video RTP packets."
+                  stream={inboundVideoRtpStats}
+                  metric="packetsLost"
+                  domain={[0, 100]}
+                  unit=" packets"
+                />
+
+                {/* Frames Per Second */}
+                <Metric
+                  title="Frames per second"
+                  description="Number of inbound video frames displayed per second."
+                  stream={inboundVideoRtpStats}
+                  metric="framesPerSecond"
+                  domain={[0, 80]}
```

Comment:
This is unchanged, but curious why 0-80 instead of the more typical 0-100 or 0-60 in the other graphs?

Reply:
We're mostly going to be around 60 fps, and just a bit of headroom looked the best.